$NetBSD$

Add yacc to list of parser generators to check for in configure phase.

--- configure.orig	2019-02-07 15:02:22.000000000 +0000
+++ configure
@@ -14959,7 +14959,7 @@ else
   RANLIB="$ac_cv_prog_RANLIB"
 fi
 
-for ac_prog in 'bison -y' byacc
+for ac_prog in 'bison -y' byacc yacc
 do
   # Extract the first word of "$ac_prog", so it can be a program name with args.
 set dummy $ac_prog; ac_word=$2
